internal/linker/link_node_modules.ts

```diff
@@ -439,21 +439,6 @@ function liftElement(element: LinkerTreeElement): LinkerTreeElement {
     return {name, link};
   }
 
-  // No link but all child elements have aligning links
```

Comment:
I think this was just an "optimization" to avoid creating unnecessary folders just to symlink 1+ child elements within those folders. So if it was just an "optimization" then it would be ok to drop it?

You're saying this could be a problem when `a` is something special such as `@bazel`?

Reply:
I guess this made would convert real directories to links which was the issue? Where the logic above this would only convert links to simpler links, not real directories to links...